torch/csrc/jit/python/python_sugared_value.cpp

```diff
@@ -548,12 +548,29 @@ std::shared_ptr<SugaredValue> BooleanDispatchValue::call(
   return value->call(loc, caller, inputs, attributes, n_binders);
 }
 
+std::shared_ptr<SugaredValue> PythonExceptionValue::call(
+    const SourceRange& loc,
+    Function& caller,
+    at::ArrayRef<NamedValue> inputs,
+    at::ArrayRef<NamedValue> attributes,
+    size_t n_binders) {
+  Value* message;
+  if (inputs.size() == 0) {
+    message = insertConstant(*caller.graph(), "", loc);
+  } else if (inputs.size() == 1) {
+    message = inputs.at(0).value(*caller.graph());
+  } else {
+    throw ErrorReport(loc) << "exceptions in TorchScript only support"
```

Comment:
Why not construct a tuple from the inputs ?

Reply:
This isn't resolved. I think an exception that has multiple inputs will be a very common case.